verilog/analysis/checkers/undersized_binary_literal_rule_test.cc

```diff
@@ -169,6 +169,46 @@ TEST(UndersizedBinaryLiteralTest, DecimalNumbersNeverCare) {
       kTestCases, "");
 }
 
+// There is no good auto-fix test infrastructure in place, so this is
+// somewhat manual.
+// TODO(hzeller): add test infrastructure in linter_test_utils.h
+
+static std::unique_ptr<UndersizedBinaryLiteralRule> CreateConfiguredRule() {
+  auto result = std::make_unique<UndersizedBinaryLiteralRule>();
+  CHECK(result->Configure("bin:true;hex:true;oct:true").ok());
+  return result;
+}
+
+static std::string UndersizeApplyFix(absl::string_view input) {
+  VerilogAnalyzer analyzer(input, "");
+  CHECK(analyzer.Analyze().ok()) << "Parse issue " << input;
```

Comment:
ASSERT_EQ, to allow other tests to run.

Reply:
Same problem here, `ASSERT_EQ()` does not work inside helper functions.